### DISCUSSION TITLE
ssk-key is right why throw exception?ror loading key "(stdin)": error in libcrypto Error: The ssh client configuration encountered an error: Command failed: ssh-

### DISCUSSION THREAD

**Discussion by lozn00** [JamesIves/github-pages-deploy-action#1777] [Q&A]
![image](https://github.com/user-attachments/assets/c901b336-3895-472f-a5c1-d0fedd63c6f6)

**Comment by JamesIves** [comment]
You need to provide more details than just a screenshot if you're asking for help here. I would need to see logs, your workflows, etc. At a glance there's an issue with your SSH configuration, [did you follow the steps in the README](https://github.com/JamesIves/github-pages-deploy-action?tab=readme-ov-file#using-an-ssh-deploy-key-)?
